### PR TITLE
Update doc link about light node.

### PR DESCRIPTION
**Pull request by dannywillems**
### Description

Update doc link. The current one redirects to the new suggested one.

### CODE REVIEW

File: packages/mobile/README.md [hunk 1]
```diff
@@ -424,7 +424,7 @@ $ adb kill-server && adb start-server
 [detox]: https://github.com/wix/Detox
 [e2e readme]: ./e2e/README.md
 [graphql code generator]: https://github.com/dotansimha/graphql-code-generator
-[light node]: https://github.com/ethereum/wiki/wiki/Light-client-protocol
+[light node]: https://eth.wiki/en/concepts/light-client-protocol
```
Comment (jmrossy) [inline packages/mobile/README.md]: This link is quite old. We should probably link to our own docs now:
https://docs.celo.org/overview#ultralight-synchronization

**Comment by jmrossy** [comment]
Thanks for the PR @dannywillems! Just one comment.

**Comment by dannywillems** [comment]
> Thanks for the PR @dannywillems! Just one comment.

changed

**Review by jmrossy — APPROVED** [review]
Thanks!

**Pull request merged**